pyaedt/modules/MaterialLib.py

```diff
@@ -477,6 +477,16 @@ def export_materials_to_file(self, full_json_path):
             
         """
         def find_datasets(d, out_list):
+            """Find datasets.
+            
+            Parameters
+            ----------
+            d :
+            
+            out_list : list
+            
+            """
+            
```

Comment:
```suggestion

```
This is a nested method and cannot be accessed from anywhere but inside the `export_materials_to_file` method, so it does not really need documented.